src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -54,6 +54,11 @@
 
 {# Widgets #}
 
+{% block form_start %}
+  {% set attr = attr|merge({'data-alerts-success': form.vars.alerts.success|default([])|length()}) %}
```

Comment:
Maybe we should add the other "scopes" at least errors and warnings?

Reply:
Im not a fan of adding unused code :smile: